config/spl-build.m4

```diff
@@ -228,7 +225,9 @@ AC_DEFUN([SPL_AC_RPM], [
 		AC_MSG_RESULT([$HAVE_RPMBUILD])
 	])
 
-	RPM_DEFINE_COMMON='--define "$(DEBUG_SPL) 1" --define "$(DEBUG_KMEM) 1" --define "$(DEBUG_KMEM_TRACKING) 1"'
+	RPM_DEFINE_COMMON='--define "$(DEBUG_SPL) 1"'
+	RPM_DEFINE_COMMON+=' --define "$(DEBUG_KMEM) 1"'
+	RPM_DEFINE_COMMON+=' --define "$(DEBUG_KMEM_TRACKING) 1"'
```

Comment:
Is this a stylistic / code cleanup change?  If so, we should make it upstream instead.

Reply:
Hmm I believe those changes are from the upstream commit https://github.com/delphix/spl/pull/3/commits/84a80d5f2d14fdfd16d440eef2e3d6e756a1165c which ended up in this PR when I opened it. Still getting familiar with the github workflow...